### PR TITLE
make Gemfile ruby declaration compatible with rvm

### PR DESCRIPTION
**Pull request by porbas**
+ update yanked mimemagic gem version

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,10 @@
 source 'https://rubygems.org'
 
 ruby '~> 2.7'
+# rvm doesn't support ruby version operators
+# but supports magic #ruby= comment
+# https://github.com/rvm/rvm/issues/3737#issuecomment-271707899
+#ruby=2.7.2
```
[the remark this answers is not in the document]
Comment (pawelpacana) [reply Gemfile:7]: why do we need to specify suggested ruby version for every possible Ruby version manager out there?

**Comment by pawelpacana** [comment]
Not trying to nitpick but this PR contains one change to merged with no questions asked, and one which I'd like to discuss why is it ever needed.

If that version declaration on Gemfile breaks something for you, I'd be more than glad to find out out how to get rid of it. It only exists because Heroku insists on it. Perhaps in 2021 Ruby version used by Heroku to build an application image can be specified differently.

File: Gemfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (porbas) [reply Gemfile:7]: I see attitude to make sample app as easy to get it running as possible: `docker-compose.yml`, `make setup`. But in the same time it is not straightforward to run it with RVM. One would get error:

     Unknown ruby interpreter version (do not know how to handle): ~>2.7.

So can we make ruby version specification compatible with Heroku AND RVM?

**Comment by porbas** [comment]
If Horoku can accept ruby version as:

    ruby '2.7.2'

Than it should good enough for average RVM user, including me :)

**Comment by pawelpacana** [comment]
> If Heroku can accept ruby version as: 2.7.2.

It does accept such strict Ruby version specification. 

The consequence of this is that you'll no longer be able run this application, tests and whatnot on Ruby with a different patch level, in development. Bundler would loudly protest. I consider this a huge friction. That reason was a major driver for relaxed version specification in the first place.

I'd be glad to know your POV on what does not work in RVM (not using it personally), I haven't learnt that from commits nor description.

Whatever the problem is we may either:
- decide on stricter Ruby version specification in hope for broader support in RVM and other popular-and-too-much-assuming-tools
- decide on no version specification in Gemfile and specifying deployment version elsewhere only for Heroku (if possible)

File: Gemfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (pawelpacana) [reply Gemfile:7]: I wasn't aware of that RVM message. Sounds odd, but I guess RVM isn't that actively maintained nowadays. It probably wan't support any other version declaration than strict, right?

[the remark this answers is not in the document]
Comment (pawelpacana) [reply Gemfile:7]: Could you please check if you still get this error when there is no version specified in Gemfile?

**Comment by porbas** [comment]
I've put info about RVM complains in conversation. Sorry to not put it in description first place.

I have no experience with other ruby version managers besides RVM. I thought everyone use strict ruby versions ;)

Third option to consider, which works for me:
* configure ruby version in `.rvm-version`
* add `.rvm-version` to .`gitignore`
* add README section for rvm users

File: Gemfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (porbas) [reply Gemfile]: RVM uses my _default_ ruby. It is OK for me.

[the remark this answers is not in the document]
Comment (pawelpacana) [reply Gemfile]: Thank you for checking 🙏

**Comment by pawelpacana** [comment]
* cherry-picked https://github.com/RailsEventStore/cqrs-es-sample-with-res/commit/fcc09088866c8d6119ee1012cbcad42fcbcabb66 with a note on missing macOS dependency
* decided to go with strict to support version equally for everyone (moving friction elsewhere) 66e22d1

**Pull request closed without merging**